docs/source/projects/architecture/content-architecture.rst

```diff
@@ -6,6 +6,15 @@ This page has information about the different architectural designs of core
 pieces in the Jupyter ecosystem. Some of these are individual projects, and others
 show the relationships between projects.
 
+Projects overview
+=================
+
+Below is a high level visual overview of project relationships. It is current as of
+2022.
+
+.. image:: /_static/_images/repos_map.png
```

Comment:
How is this image being generated? Do we have docs about it somewhere so that somebody else could make edits? Could we provide a URL link to instructions as a comment or something?

Reply:
Not in this PR, but I'd suggest that in the future we use [excalidraw](https://excalidraw.com/) for these kinds of drawings. This would let us export the drawing as an SVG with the ExcaliDraw metadata embedded inside it, so a maintainer could make edits by loading the SVG back into excalidraw, adding it back here, and then making a PR to update the SVG.

I'll open an issue to track this, just noting it here.

Reply:
This image is generated with [drawio](https://app.diagrams.net/), the scheme is embedded in both images (png and svg), so anyone can edit it.

Reply:
Is that documented somewhere? Maybe I missed it, anyway as I say it doesn't need to block the PR